cmd/keeper/cmd/keeper.go

```diff
@@ -1391,6 +1482,10 @@ func (p *PostgresKeeper) postgresKeeperSM(pctx context.Context) {
 	targetRole := db.Spec.Role
 	log.Debugw("target role", "targetRole", string(targetRole))
 
+	// Set metrics to power alerts about mismatched roles
```

Comment:
These metrics are particularly useful to power an up-to-date view of keeper state. It looks like this in a dashboard:

<img width="765" alt="Screenshot 2019-04-30 at 17 01 41" src="https://user-images.githubusercontent.com/3518874/56975899-a6760000-6b69-11e9-9801-9f008be660bd.png">